packages/text-annotator-react/src/TextAnnotationPopup/TextAnnotationPopup.tsx

```diff
@@ -87,12 +93,6 @@ export const TextAnnotationPopup = (props: TextAnnotationPopupProps) => {
     whileElementsMounted: autoUpdate
   });
 
-  const dismiss = useDismiss(context);
-
-  const role = useRole(context, { role: 'dialog' });
-
-  const { getFloatingProps } = useInteractions([dismiss, role]);
```

Comment:
@oleksandr-danylchenko I noticed that `getFloatingProps` isn't actually used. Is this an omission? Or is this no longer needed now we're using the FloatingPortal & FloatingFocusManager?